docs/hand_wire.md

```diff
@@ -169,34 +253,52 @@ Before beginning to solder, it helps to have your wire pre-bent (if using single
 
 If you're not using any insulation, you can try to keep the column wires elevated, and solder them near the tips of the keyswitch contacts - if the wires are sturdy enough, they won't short out to the row wiring an diodes.
 
-## Wiring Things to the Teensy
+</details>
+
+# Wiring up the controller
+
```

Comment:
A section for existing firmware might be a nice to have here, if someone is building an existing handwire

Reply:
I added this to the start of "planning the matrix"